pkg/connector/group.go

```diff
@@ -228,7 +244,16 @@ func (o *groupResourceType) Grants(
 				return nil, "", nil, err
 			}
 
-			rv = append(rv, roleGroupGrant(groupID, roleResourceVal))
+			groupTrait, err := sdkResource.GetGroupTrait(resource)
+			if err != nil {
+				return nil, "", nil, fmt.Errorf("okta-connectorv2: failed to get group trait: %w", err)
+			}
+			usersCount, ok := sdkResource.GetProfileInt64Value(groupTrait.Profile, usersCountProfileKey)
+			if !ok {
+				return nil, "", nil, fmt.Errorf("okta-connectorv2: failed to get groups users count")
```

Comment:
Could this cause existing syncs that began with an older version of the connector to error when they pick up these changes?